app/views/layouts/_flash.html.erb

```diff
@@ -1,14 +1,14 @@
 <% if flash.present? %>
-    <div class="row-fluid alerts">
-      <% flash.each do |name, message| %>
-          <div class="alert fade in <%= flash_alert_class(name) %>">
```

Comment:
For some reasons I don't see any animation. But `fade` class add `opacity: 0` property to the `div` so you can't see the alert. 
![screen shot 2561-06-07 at 11 24 26](https://user-images.githubusercontent.com/3861498/41078393-c3baf690-6a46-11e8-9804-fb888626760e.png)
![screen shot 2561-06-07 at 11 24 31](https://user-images.githubusercontent.com/3861498/41078394-c3fb790e-6a46-11e8-92fe-32374bf5d007.png)

Reply:
@vvmarulin did you include the JS that comes with BS? The animation is not handled by CSS alone.

Reply:
Ok. I check it one more time. Animation works. This [line](https://github.com/lockstep/rails_new/blob/master/app/assets/javascripts/application.js#L17) add the JS that comes with BS. 

But to make this fading animation work correctly we must use `show` class instead of `in` ([documentation](https://getbootstrap.com/docs/4.0/components/alerts/#dismissing)). Also it's required to add `alert-dismissible` class to make alert dismissible. But in real this class just change stylings a bit and that's all.

But I still voting for removing fade animation. I thought this template should be styled (so it's not ugly) but simple as much as possible.

Reply:
@vvmarulin this is strange since I remember I fixed this in the past in https://github.com/lockstep/rails_new/pull/44. Did it get rebased-out @citizen428?

Reply:
I have no strong opinions re the fade. As for where your changes went @janklimo, I'm not really sure. Ask the cat:

![eated_it](https://user-images.githubusercontent.com/47985/41082933-17aac0ba-6a5a-11e8-9b9c-1735292fed77.jpg)